createdisk.sh

```diff
@@ -33,12 +33,21 @@ function create_disk_image {
 # CRC_VM_NAME: short VM name to use in crc_libvirt.sh
 # VM_PREFIX: full VM name with the random string generated by openshift-installer
 CRC_VM_NAME=${CRC_VM_NAME:-crc}
+VIRT_SPARSIFY=${VIRT_SPARSIFY:virt-sparsify}
```

Comment:
needs to have a '-' after the ':'
`VIRT_SPARSIFY=${VIRT_SPARSIFY:-virt-sparsify}`
and a similar line must be present for ${QEMU_IMG}